scripts/export_tasks.py

```diff
@@ -0,0 +1,40 @@
+import ast
```

Comment:
Maybe add a short docstring at the top explaining what this does/what's the output of the script

Reply:
Also mention the limitation that it does not include transformers